pkg/platform/kube/platform.go

```diff
@@ -1173,6 +1198,20 @@ func (p *Platform) GetNamespaces(ctx context.Context) ([]string, error) {
 	return namespaceNames, nil
 }
 
+func (p *Platform) GetFunctionScrubber() *functionconfig.Scrubber {
```

Comment:
I guess a similar function will be needed for API gateway scrubber as you can also create ApiGW via nuctl (same as you did with the function scrubber)